### DISCUSSION TITLE
Multiple MIDI channels via USB

### DISCUSSION THREAD

**Discussion by anycolorsmith** [probonopd/MiniDexed#243] [Q&A] [answered]
Hello, friends! Can i send several channels to Minidexed to play various voices? For example from Arturia keystep sequenser?

**Comment by probonopd** [comment, marked as answer]
Sure. Just use different MIDI channels.

**Comment by JoaquinSainz** [comment]
Yes you can, each sound module can be playing different sounds controlling them on different MIDI channels.

**Reply by anycolorsmith** [reply]
> Yes you can, each sound module can be playing different sounds controlling them on different MIDI channels.

Thanx!

**Reply by izash** [reply]
Sure:

https://fb.watch/d8UDUwwgPQ/
